target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-rutx11.dts

```diff
@@ -0,0 +1,210 @@
+			};
+		};
+
+		mdio@90000 {
+			status = "ok";
+			pinctrl-0 = <&mdio_pins>;
+			pinctrl-names = "default";
+			phy-reset-gpio = <&tlmm 62 0>;
+
+			ethphy4: ethernet-phy@4 {
+				qcom,fiber-enable;
+			};
+		};
+
+		pinctrl@1000000 {
+			mdio_pins: mdio_pinmux {
+				mux_1 {
+					pins = "gpio53";
+					function = "mdio";
+					bias-pull-up;
+				};
+				mux_2 {
+					pins = "gpio52";
+					function = "mdc";
+					bias-pull-up;
+				};
+			};
+		};
+
+		gpio_keys {
+			compatible = "gpio-keys";
+
+			reset {
+				label = "reset";
+				gpios = <&tlmm 4 1>;
+				linux,code = <0x198>;
+			};
+		};
+		
+		usb3: usb3@8af8800 {
+			dwc3@8a00000 {
+				snps,dis_u2_susphy_quirk;
+				snps,dis_u3_susphy_quirk;
+			};
+		};
+
+		usb2: usb2@60f8800 {
+			dwc3@6000000 {
+				snps,dis_u2_susphy_quirk;
+			};
+		};
+
+		gpio-export {
+			compatible = "gpio-export";
+			#size-cells = <0>;
+
+			gpio_modem_reset {
+				gpio-export,name = "modem_reset";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 21 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_modem_power {
+				gpio-export,name = "modem_power";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 20 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_modem2_reset {
+				gpio-export,name = "modem2_reset";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 13 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_modem2_power {
+				gpio-export,name = "modem2_power";
+				gpio-export,output = <0>;
+				gpios = <&stm32_io 14 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_sim_select {
+				gpio-export,name = "sim_sel";
+				gpio-export,output = <1>;
+				gpios = <&stm32_io 22 GPIO_ACTIVE_LOW>;
+			};
+
+			gpio_out_1 {
+				gpio-export,name = "gpio23";
+				gpio-export,output = <0>;
+				gpio-export,direction_may_change;
+				gpios = <&stm32_io 23 GPIO_ACTIVE_HIGH>;
+			};
+
+			gpio_in_1 {
+				gpio-export,name = "gpio24";
+				gpio-export,input = <0>;
+				gpio-export,direction_may_change;
+				gpios = <&stm32_io 24 GPIO_ACTIVE_LOW>;
+			};
+		};
+
+		leds {
+			compatible = "gpio-leds";
+
+			led_wifi2g {
+				label = "wifi_2g";
+				gpios = <&stm32_io 19 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_wifi5g {
+				label = "wifi_5g";
+				gpios = <&stm32_io 18 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_sim1 {
+				label = "sim_1";
+				gpios = <&stm32_io 0 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_sim2 {
+				label = "sim_2";
+				gpios = <&stm32_io 1 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_wan_wifi {
+				label = "wan_wifi";
+				gpios = <&stm32_io 3 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_wan_eth {
+				label = "wan_eth";
+				gpios = <&stm32_io 2 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_wan_2g {
+				label = "wan_2g";
+				gpios = <&stm32_io 4 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_wan_3g {
+				label = "wan_3g";
+				gpios = <&stm32_io 5 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_wan_4g {
+				label = "wan_4g";
+				gpios = <&stm32_io 6 GPIO_ACTIVE_HIGH>;
+			};
+
+			led_ssid_1 {
+				label = "ssid_1";
+				gpios = <&stm32_io 7 GPIO_ACTIVE_HIGH>;
+				linux,default-trigger = "timer";
```

Comment:
What’s this for?

Same for the other 4 “ssid” but probably really RSSI LEDs.

Reply:
Replaced all SSID labels with correct ones.

Reply:
This question wasn’t about the “SSID” name, it was about this specific line:

```
				linux,default-trigger = "timer";
```

> What’s this for?

Reply:
There’s a standard way to use LED to indicate boot status in OpenWrt, via `/etc/diag.sh`. You can set some `led-*` aliases in the `.dts` file to enable this behavior (`led-boot`, `led-failsafe`, `led-running`, `led-upgrade`). Have a look at some of the neighboring `.dts` files to see how this works.

Reply:
Added boot behaviour for rssi low led.